server/schema/giftcard/res.js

```diff
@@ -12,6 +12,22 @@ const resolvers = {
       _authAdmin(loggedInUser)
       return genGiftCard(input.amount, input.expiry)
     },
+    async deleteGiftCard(_, {input}, {loggedInUser}) {
+      _authAdmin(loggedInUser)
+      return await GiftCard.destroy({
+        where: {
+          id: {
+            $in: input,
+          },
+        },
+      })
+    },
+    async createGiftCard(_, {input}, {loggedInUser}) {
+      _authAdmin(loggedInUser)
+      return await GiftCard.create(input).then(async function(giftCard) {
```

Comment:
.then(async function(giftCard) {  ...  -> .then( giftCard=>giftCard )

why need to put async function inside then() ?